game/scripts/vscripts/libraries/chatcommand.lua

```diff
@@ -120,11 +120,20 @@ function ChatCommand:OnPlayerChat(keys)
 
 	  -- Remove fog of war on the map, revealing everything
 	  elseif string.find(text, "-nofog") then
-	    mode:SetFogOfWarDisabled(true)
+	    GameRules:GetGameModeEntity():SetFogOfWarDisabled(true)
 
 	  -- Bring back the fog of war
 	  elseif string.find(text, "-fog") then
-	    mode:SetFogOfWarDisabled(false)
+	    GameRules:GetGameModeEntity():SetFogOfWarDisabled(false)
+
+	  -- Bring back the fog of war
+	  elseif string.find(text, "-fixspawn") then
```

Comment:
It was added by accident like the bot stuff, I think those changes are already merged previously.